utils/constants.jsx

```diff
@@ -138,6 +138,8 @@ export const ActionTypes = keyMirror({
 
     BROWSER_CHANGE_FOCUS: null,
 
+    LOADED_PLUGIN: null,
+    UNLOADED_PLUGIN: null,
```

Comment:
this comment can be disregarded in light of https://github.com/mattermost/mattermost-webapp/pull/5262#discussion_r413503953 since this will be removed.